selftests/unit/test_test.py

```diff
@@ -81,12 +94,22 @@ def check(uid, name, variant, exp_logdir):
 
         self.assertEqual(os.path.basename(tst.workdir),
                          os.path.basename(tst.logdir))
-        flexmock(tst)
-        tst.should_receive('filename').and_return(os.path.join(self.tmpdir,
-                                                               "a"*250))
-        self.assertEqual(os.path.join(self.tmpdir, "a"*250 + ".data"),
+
+    def test_data_dir(self):
+        """
+        Tests that that a valid datadir exists follwowing the test filename
```

Comment:
I don't think the double `that` was intentional, right?

Reply:
nope, fixing it.